### DISCUSSION TITLE
Found A Bug - Edit Repository Details

### DISCUSSION THREAD

**Discussion by kmanadkat** [community/community#62164] [Repositories]
### Select Topic Area

Bug

### Bug Description
I am unable to remove `Releases`, `Packages`, `Environments` from my [public repository](https://github.com/kmanadkat/advanced-react-patterns-project/tree/master) using `Edit repository details` Modal.

### Machine Info
- Mac OS - Ventura (13.4.1)
- Chrome Version 114.0.5735.90 (Official Build) (arm64)
- Browser Extensions - [React Developer Tools](https://chrome.google.com/webstore/detail/fmkadmapgofadopljbjfkapdkoienihi) & [Redux DevTools](https://github.com/reduxjs/redux-devtools)

### Steps to Reproduce 
Created a very short screen recording, PFA

https://github.com/community/community/assets/25281055/be0ec017-39a8-447b-a76b-e1f7345b2856

> I don't believe this has to do anything with extensions or Chrome. BE service for updating the repository config might be down, but in that case I should not see success alert after hitting Save Changes button. I also tried refreshing the browser few times if there is some cache issue, but that didn't solve it.

**Comment by priyankarpal** [comment]
@kmanadkat I also got the same bug. 

- Kali Linux
- Firefox

**Comment by priyankarpal** [comment]
@Iron-Ham please check

**Comment by StevenEWright** [comment]
Dupe of: https://github.com/orgs/community/discussions/62025
